repository-data/webfiles/src/main/resources/site/freemarker/intranet/news.ftl

```diff
@@ -1,2 +0,0 @@
-<#ftl output_format="HTML">
```

Comment:
Is this not used at all?

Reply:
it was the dummy template I added, joe renamed to news internal